src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -34,40 +35,17 @@ public HttpApiHandler(DurableTaskExtension config, TraceWriter traceWriter)
             string instanceId,
             OrchestrationClientAttribute attribute)
         {
-            if (this.config.NotificationUrl == null)
-            {
-                throw new InvalidOperationException("Webhooks are not configured");
-            }
-
-            Uri notificationUri = this.config.NotificationUrl;
+            GetClientResponseLinks(request, instanceId, attribute, out var statusQueryGetUri, out var sendEventPostUri,
```

Comment:
style nit: I prefer all parameters be on the same line or all of them are on their own line. Same for here and below.